src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1053,6 +1053,10 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
             $realDefinition = new Workflow\Definition($places, $trs, $initialMarking);
             $validator->validate($realDefinition, $name);
 
+            foreach ($workflow['definition_validators'] as $validatorClass) {
+                (new $validatorClass())->validate($realDefinition, $name);
```

Comment:
The DefinitionValidatorInterface does not guarantee that the class has a constructor that can be called without any argument.